pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -100,6 +92,11 @@ func (c *Cluster) checkHealth() error {
 	c.acquireOrchestrationLock()
 	defer c.releaseOrchestrationLock()
 
+	if c.spec.Mon.Count == 0 || c.ClusterInfo == nil {
+		logger.Warningf("skipping mon health check since cluster info is not initialized")
```

Comment:
The message is not 100% accurate since it's either the mon count is 0 or clusterInfo is nil

Reply:
I'll pick up the changes from #5869